lib/level.js

```diff
@@ -10,7 +11,6 @@ class Level {
 	}
 
 	endGame() {
-		// remove the keyup event listener-need to name it and move it to do this.
```

Comment:
have you played through to the end to see if one can still rack up points after the game is over? I don't think so, but not positive because I can't get to the end!=}